sys-apps/plocate/plocate-1.1.5.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-info meson systemd
+
+DESCRIPTION="Posting locate is a much faster locate"
+HOMEPAGE="https://plocate.sesse.net/"
+
+if [[ "${PV}" == "9999" ]]; then
```

Comment:
But you're not committing a live ebuild. :)

(I'm fine with it if you are, in this case.)

Reply:
Upstream makes releases often enough that we probably won't need a live ebuild - removed.